test/get_filter.tcl

```diff
@@ -0,0 +1,31 @@
+# Read in design and libraries
+read_liberty asap7_invbuf.lib.gz
```

Comment:
I pushed an abbreviated asap7 library that will read faster for reg1_asap7.v
test/asap7_small.lib.gz